packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6108,6 +6111,15 @@ class Listener extends SingleChildRenderObjectWidget {
   /// no longer directed towards this receiver.
   final PointerCancelEventListener? onPointerCancel;
 
+  /// Called when a pan/zoom begins such as from a trackpad gesture
```

Comment:
Also, I mentioned this elsewhere, but this also includes rotation, right?

Otherwise to me, pan/zoom seems like pan or zoom, and pan-zoom seems like pan and zoom.  In my testing on my Macbook, it seems like a 2 finder pan gesture is treated separately from a pinch-to-zoom, so maybe pan/zoom is most accurate?  See my discussion about combining gestures like this in my main review comment.

Reply:
Also, add periods at the end of these 3 comments.